src/OpenIddict.Abstractions/Resources/xlf/OpenIddictResources.tr.xlf

```diff
@@ -1,600 +1,600 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
 <xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
   <file datatype="xml" source-language="en" target-language="tr" original="../OpenIddictResources.resx">
     <body>
       <trans-unit id="ID3000">
         <source>The security token is missing.</source>
-        <target state="new">The security token is missing.</target>
+        <target state="new">Güvenlik belirteci eksik.</target>
```

Comment:
Can you please change the `state` of all the translated strings to `translated`?

Reply:
Ok, I committed states.